### ISSUE TITLE
Image width height sample xy coordinates 

### ISSUE THREAD

**Issue by cpetro** [Pewww/check-light-or-dark#4]
hey sorry to bug you , I have implemented this plugin in a project I am working on and really appreciate it because it is exactly the functionality that I need , but I have some problem with the sample coordinates . 

if I provide an image ,and some coordinates, it doesn't give the desired result , it just makes them all dark . If I use the default setting x : 0 , y: 0 , it works tho  

I am expecting that this is based on the natural image dimensions and think there is a problem with the coordinates or some sort of problem with the plugin, also the image that I am using could be a background image or could be object-fit:cover so I understand that its not relating to the image as its displayed in the site , but still it seems not to get the sample of the top right corner or bottom left corner, 

say for instance :

```
let panel = document.querySelector('.panel');
let image = panel.querySelector('.image');

if( image ) {
  let width = image.naturalWidth;
  let height = parseInt(image.naturalHeight;

  // bottom left corner 100 px square sample 
  lightOrDarkImage({
    image: image.src,
    width: 100,
    height: 100,
    x: 0,
    y: (height - 100 ),
  }).then(result => {
    panel.classList.add(result + '-bottom-left');
  });

  // top right corner 100 px square sample
  lightOrDarkImage({
    image: image.src,
    width: 100,
    height: 100,
    x: (width - 100),
    y: 0
  }).then(result => {
    panel.classList.add(result+ '-top-right');
  });
}

```

am I mistaking something here ?  Thanks !

**Comment by Pewww** [comment]
@cpetro Hi ceptro, thank you for issuing!

I have a busy schedule today, so I think I can check it tomorrow🙏🏻

It you don't mind, could you share the image that is causing the problem?

**Comment by cpetro** [comment]
Hi ! 

I made an example for testing :

https://codepen.io/chrispetro/pen/LYmZjgY

**Comment by Pewww** [comment]
@cpetro I will try to check and solve it ASAP.
Thank you for adding example!

**Comment by Pewww** [comment]
@cpetro Fixed in `v1.0.5`!
I also added 'neutral' result when images's lightness and darkness degree are same.
https://www.npmjs.com/package/@check-light-or-dark/image/v/1.0.5

If it's resolved, please close the issue or leave a comment.
Thanks :D

**Comment by cpetro** [comment]
Thanks ! :D
